cmd/root.go

```diff
@@ -164,3 +145,35 @@ func recursiveAutoGenDisable(cmd *cobra.Command) {
 		recursiveAutoGenDisable(child)
 	}
 }
+
+func printErrorWithHints(c *config.Config, cmd *cobra.Command, err error) {
+	var connectErr *connect.Error
+	if !errors.As(err, &connectErr) || connectErr.Code() != connect.CodeUnauthenticated {
+		cmd.PrintErrln(err)
+		return
+	}
+
+	token := c.GetToken()
+	if token != "" {
```

Comment:
Potential swallow of printing any error.

Reply:
Done!